src/mosaic/MosaicPage.js

```diff
@@ -27,10 +27,18 @@ class MosaicPage extends Component {
 
   render() {
     const { isFetching, tracks, error } = this.props;
-    const albumArtTiles = tracks.map(t => ({
-      url: `${t.album.images[1].url}?${new Date().getTime()}`,
+    const albumArtTiles = tracks.map(t => {
+      let url = "";
+      try{
```

Comment:
I would've liked to have used a ternary here, but I think an error would throw because it's in a template string? I could be wrong though.